src/QMCApp/QMCDriverFactory.cpp

```diff
@@ -247,7 +252,7 @@ std::unique_ptr<QMCDriverInterface> QMCDriverFactory::createQMCDriver(xmlNodePtr
   if (das.new_run_type == QMCRunType::VMC || das.new_run_type == QMCRunType::CSVMC)
   {
     //VMCFactory fac(curQmcModeBits[UPDATE_MODE],cur);
-    VMCFactory fac(das.what_to_do.to_ulong(), cur);
+    VMCFactory fac(das.what_to_do[UPDATE_MODE], cur);
```

Comment:
Is passing just the UPDATE_MODE bit the same as passing the entire bitset?   The  VMCFactory code appears to use the GPU_MODE bit for selecting the CUDA version.

Reply:
This will be addressed in PR's to follow